data/data-pipeline/conftest.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    """Adds --update_snapshots as an optional flag that can be parsed from the
```

Comment:
Ahh that's good to know! Maybe also a sign that I should switch to a different IDE or at least configure some better extensions for Atom, because I haven't encountered that feature yet 😅

Reply:
Haha perhaps! I really enjoy PyCharm (which is free, but not great for full stack because it doesn't have support for many other languages), but most of the folks on the team are on VSCode.